.github/workflows/ci.yml

```diff
@@ -24,7 +24,6 @@ jobs:
       - name: 'Authenticate with GCP'
         uses: google-github-actions/auth@v2
         with:
-          workload_identity_provider: ${{ vars.GCP_WORKLOAD_IDENTITY_PROVIDER }}
```

Comment:
why this change?

Reply:
It was a change I did to see if the CI could work again, but I already reverted it.
GitHub seems to have some problems:
![image](https://github.com/user-attachments/assets/607ba174-9015-45e8-ab04-a5fd8e7cd8b1)